docker-compose.yml

```diff
@@ -18,8 +18,16 @@ services:
 
   cvat_redis:
     container_name: cvat_redis
-    image: redis:7.0-alpine
+    image: eqalpha/keydb:x86_64_v6.3.2
     restart: always
+    command: [
+      "keydb-server",
+      "/etc/keydb/keydb.conf",
+      "--storage-provider", "flash", "/data/flash",
```

Comment:
Isn't FLASH support an enterprise-only feature? It says so here: <https://docs.keydb.dev/docs/config-file/>.

Reply:
https://docs.keydb.dev/blog/2020/01/20/blog-post/
Flash is an available feature for the open source version: https://github.com/Snapchat/KeyDB#building-keydb and is enabled by default for provided docker images https://github.com/Snapchat/KeyDB/blob/main/pkg/docker/Dockerfile#L56.
I don't know why the example config still has this note in it